core/server/data/schema.js

```diff
@@ -161,12 +162,23 @@ var db = {
             uuid: {type: 'string', maxlength: 36, nullable: false},
             name: {type: 'string', maxlength: 150, nullable: false, unique: true},
             slug: {type: 'string', maxlength: 150, nullable: false, unique: true},
-            secret: {type: 'string', maxlength: 150, nullable: false, unique: true},
+            secret: {type: 'string', maxlength: 150, nullable: false},
```

Comment:
No worries, we added this functionality sometime in the past to add uniqueness. Really handy to have it now :).